### ISSUE TITLE
Detect failing workers

### ISSUE THREAD

**Issue by benoit74** [openzim/zimfarm#876]
We should not let a failing worker (considering that all its tasks are failing early) to continue to get new tasks

**Comment by rgaudin** [comment]
Surprised we did not have a ticket for that ; we discussed it recently

**Comment by benoit74** [comment]
Yep, we discussed it in #857 and agreed it should make its way to is own ticket but I realized this morning it never did.

**Comment by benoit74** [comment]
Can you have a look at this issue and propose something to implement here?

**Comment by benoit74** [comment]
Probably to be handled together with https://github.com/openzim/zimfarm/issues/1022, a worker could be `cordon` either from its local config or from a central system (admin user or automated detection of failures); probably needs two distinct status in DB for clarity (one for central and one for local).
